tasks/main.yml

```diff
@@ -57,7 +63,8 @@
       - "--token '{{ az_devops_accesstoken }}'"
       - "--runasservice"
     build_agent_cmd_args:
-      - "--pool '{{ az_devops_agent_pool_name }}'"
+      - "--deploymentpool"
+      - "--deploymentpoolname '{{ az_devops_agent_pool_name }}'"
```

Comment:
These are already specified in the `deployment_pool_cmd_args`.  Shouldn't need them here.

Reply:
they are not,
probably confused with deploymentgroup and deploymentgroupname